### PR TITLE
Cloud pipeline monitoring

### PR DESCRIPTION
**Pull request by idomic**

### CODE REVIEW

File: .github/workflows/ci-mac.yml [hunk 1]
```diff
@@ -82,6 +82,7 @@ jobs:
     - name: Run tests
       env:
         PLOOMBER_STATS_ENABLED: false
+        CI_CLOUD_API_KEY: ${{ secrets.CI_CLOUD_API_KEY }}
```
Comment (edublancas) [inline .github/workflows/ci-mac.yml:85]: I think this should have a more specific name, maybe `PLOOMBER_CLOUD_API_KEY`? and looks like this is also missing in the cloud docs

File: src/ploomber/cli/ploomber_io.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def wrapper(catch_exception=True, **kwargs):
     return wrapper
 
 
-# FIXME: capture only certain types of exceptions. If it's something we dind't
+# FIXME: capture only certain types of exceptions. If it's something we didn't
```
Comment (edublancas) [inline src/ploomber/cli/ploomber_io.py:65]: why was this renamed from io -> ploomber_io?

**Review by edublancas — CHANGES_REQUESTED** [review]
just a few comments

File: src/ploomber/cli/ploomber_io.py (hunk 2 above)
Comment (edublancas) [reply src/ploomber/cli/ploomber_io.py:65]: this is causing some merge conflicts so if there's isn't a strong reason to rename i'd say not to. but if we want to rename, it should be something that doesn't have "ploomber" as the module name, that's a bit weird

File: doc/cloud/api-key.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
+
+Install the Cloud branch
+========================
+To get this private beta functionality you'll have to install the private beta branch.
+To do so you can run this command:
+
+.. code-block:: console
+
+    pip install git+https://github.com/ploomber/ploomber@cloud
```
Comment (edublancas) [inline doc/cloud/api-key.rst:9]: pls remove this, no longer needed

File: doc/cloud/api-key.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
+
+Install the Cloud branch
```
Comment (edublancas) [inline doc/cloud/api-key.rst:2]: let's mention this is a beta feature and the API may change in the future

File: src/ploomber/cli/ploomber_io.py (hunk 2 above)
Comment (idomic) [reply src/ploomber/cli/ploomber_io.py:65]: This was a pycharm bug if I recall correctly. We can keep it as is to avoid the conflicts

File: .github/workflows/ci-mac.yml (hunk 1 above)
Comment (idomic) [reply .github/workflows/ci-mac.yml:85]: This is the key that gets from git secrets to run the CI why does it needs to be in the docs?

Comment (edublancas) [reply .github/workflows/ci-mac.yml:85]: because I see value in having this for end-users as well, give them the option to use and env variable to set the key

File: src/ploomber/cli/ploomber_io.py (hunk 2 above)
Comment (edublancas) [reply src/ploomber/cli/ploomber_io.py:65]: what error? renaming is going to cause conflicts as well. there are other branches that have updated this module

File: .github/workflows/ci-mac.yml (hunk 1 above)
Comment (idomic) [reply .github/workflows/ci-mac.yml:85]: I hear you, we can change the naming here, having an env var is another feature regardless to this testing var.

File: src/ploomber/cli/ploomber_io.py (hunk 2 above)
Comment (edublancas) [reply src/ploomber/cli/ploomber_io.py:65]: oh and I just remembered that the `io` module is part of the public API, if we change this, we'll break the code of some users
